src/VocabGamePlay.jsx

```diff
@@ -1,48 +1,62 @@
 import React, { Component } from 'react';
 import "./VocabGamePlay.scss";
-import {GameDataset} from "./VocabGameData.js";
+import { GameDataset } from "./VocabGameData.js";
+import WordCard from './WordCard';
 
 class VocabGamePlay extends Component {
-    constructor(props) {
-        super(props);
-        this.state = {
-            firstSelection: null,
-            secondSelection: null,
-            vocabDataSet: GameDataset,
-        }
+  constructor(props) {
+    super(props);
+    this.state = {
+      firstSelection: null,
+      secondSelection: null,
+      vocabDataSet: GameDataset,
     }
-    handleClick(e, data) {
-      console.log(data);
-      data.isSelected = !data.isSelected;
-      if (this.state.firstSelection === null) {
+  }
+
+  handleClick(e, data) {
+    console.log(data);
+    data.isSelected = !data.isSelected;
+    if (this.state.firstSelection === null) {
+      this.setState({ firstSelection: data });
+    } else if (this.state.secondSelection === null) {
+      if (data === this.state.firstSelection) {
         this.setState({ firstSelection: data });
+        return;
       }
-      else if (this.state.secondSelection === null) {
-        if (data === this.state.firstSelection) {
-          this.setState({ firstSelection: data });
-          return;
-        }
-        this.setState({ secondSelection: data }, () => { this.checkMatch(this.state.firstSelection, this.state.secondSelection) });
-      }
-    }
-    checkMatch(first, second){
-       this.isCorrectUpdate(first.word===second.translation);
+      this.setState({ secondSelection: data }, () => { this.checkMatch(this.state.firstSelection, this.state.secondSelection) });
     }
+  }
+  
```

Comment:
Okay so - we don't seem to be using event (e) at all in this function, and I assume we should be?

Is `data.isSelected = !data.isSelected` to handle if a user deselects the first card?

Again I'm a bit confused on "data" and where it comes from/how it's being used. Is it just a placeholder at the moment?

Are we structuring the if statement based on priority/order (like if it matches the first one then it doesn't go onto the second)? Because for instance, if both firstSelection and secondSelection are null like when we start then both of the if/elseif's right now would be true.